source/index.html.md

```diff
@@ -154,6 +154,7 @@ error categories, the last two digits define specific errors.
 * 0209: Two Factor authentication required
 * 0210: Device authentication required
 * 0211: Device authentication failed (Invalid IP or expired Token)
+* 0213: IP address is not allowed to access this resource
```

Comment:
Please add what @thenewversion asked for.

Reply:
Updated @viccomx